src/Mapbender/PrintBundle/Component/PrintService.php

```diff
@@ -612,29 +612,33 @@ protected function addCoordinates($pdf, $template, $jobData)
         }
 
         $efData = $jobData['extent_feature'];
+        $minX = min(array_map(function($coordinate){ return $coordinate["x"]; },$efData));
+        $minY = min(array_map(function($coordinate){ return $coordinate["y"]; },$efData));
+        $maxX = max(array_map(function($coordinate){ return $coordinate["x"]; },$efData));
+        $maxY = max(array_map(function($coordinate){ return $coordinate["y"]; },$efData));
         $fieldDataMapping = array(
```

Comment:
Min / max of the coordinates _do_ _not_ produce the bottom left / top right corner of the printed map area. Especially not for a rotated print.